inc/shortcodes/class-googledocs.php

```diff
@@ -0,0 +1,245 @@
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => esc_html__( 'Google Docs', 'shortcake-bakery' ),
+			'listItemImage'  => '<img src="' . esc_url( SHORTCAKE_BAKERY_URL_ROOT . 'assets/images/svg/icon-googledocs.svg' ) . '" />',
+			'attrs'          => array(
+				array(
+					'label'        => esc_html__( 'Document Type', 'shortcake-bakery' ),
+					'attr'         => 'type',
+					'type'         => 'select',
+					'options'      => array(
+						'document'      => esc_html__( 'Document', 'shortcake-bakery' ),
+						'spreadsheet'   => esc_html__( 'Spreadsheet', 'shortcake-bakery' ),
+						'presentation'  => esc_html__( 'Presentation', 'shortcake-bakery' ),
+						'form'          => esc_html__( 'Form', 'shortcake-bakery' ),
+						'map'           => esc_html__( 'Map', 'shortcake-bakery' ),
+					),
+					'description'  => esc_html__( 'Type of document to embed', 'shortcake-bakery' ),
+				),
+				array(
+					'label'        => esc_html__( 'URL', 'shortcake-bakery' ),
+					'attr'         => 'url',
+					'type'         => 'text',
+					'description'  => esc_html__( 'Full document URL', 'shortcake-bakery' ),
+				),
+
+				array(
+					'label'        => esc_html__( 'Height', 'shortcake-bakery' ),
+					'attr'         => 'height',
+					'type'         => 'number',
+					'description'  => esc_html__( 'Pixel height of the iframe.', 'shortcake-bakery' ),
+				),
+				array(
+					'label'        => esc_html__( 'Width', 'shortcake-bakery' ),
+					'attr'         => 'width',
+					'type'         => 'number',
+					'description'  => esc_html__( 'Pixel width of the iframe.', 'shortcake-bakery' ),
+				),
+				array(
+					'label'        => esc_html__( 'Disable Responsiveness', 'shortcake-bakery' ),
+					'attr'         => 'disableresponsiveness',
+					'type'         => 'checkbox',
+					'description'  => esc_html__( 'By default, height/width ratio of the embedded document will be maintained regardless of container width. Check this to keep constant height/width.', 'shortcake-bakery' ),
+				),
+
+				/* Options specific to "spreadsheet" document type */
+				array(
+					'label'        => esc_html__( 'Display spreadsheet header rows?', 'shortcake-bakery' ),
+					'attr'         => 'headers',
+					'type'         => 'checkbox',
+				),
+
+				/* Options specific to "presentation" document type */
+				array(
+					'label'        => esc_html__( 'Autostart?', 'shortcake-bakery' ),
+					'attr'         => 'start',
+					'type'         => 'checkbox',
+				),
+				array(
+					'label'        => esc_html__( 'Loop?', 'shortcake-bakery' ),
+					'attr'         => 'loop',
+					'type'         => 'checkbox',
+				),
+				array(
+					'label'        => esc_html__( 'Delay between slides (ms)', 'shortcake-bakery' ),
+					'attr'         => 'delayms',
+					'type'         => 'number',
+					'default'      => 3000,
+				),
+				array(
+					'label'        => esc_html__( 'Allow fullscreen mode?', 'shortcake-bakery' ),
```

Comment:
shrug.. I don't know? I just figured it was an option in the embed code, so I'd expose it here as well.